gatekeeper/namespaced/upstream/gatekeeper-patch.yaml

```diff
@@ -10,22 +10,25 @@ spec:
           args:
             # Args from the upstream
             - --audit-interval=60
-            - --log-level=WARNING
+            # --log-level=INFO
             - --constraint-violations-limit=20
             - --audit-from-cache=false
-            - --audit-chunk-size=0
+            - --audit-chunk-size=500
             # --audit-match-kind-only=false
             - --emit-audit-events=false
             - --operation=audit
             - --operation=status
+            # --operation=mutation-status
             - --logtostderr
             - --health-addr=:9090
             - --prometheus-port=8888
-            # Args added/edited by us to reduce memory footprint by only
-            # requesting kinds we have constraints for and limting the chunk
-            # size.
+            - --enable-external-data=false
+            # Args added/edited by us
+            # * Reduce log noise
+            # * Reduce memory footprint by only requesting kinds we have
+            #   constraints for and disabling mutation
+            - --log-level=WARNING
```

Comment:
kind of odd that this flag is also commented out above?

Reply:
I'm following the existing style, of matching all the upstream flags to make comparing easier, and then adding/editing the difference below.